### DISCUSSION TITLE
I can push my mobile field data to QFieldCloud, but can't synchronize to my main machine

### DISCUSSION THREAD

**Discussion by jsmith2-wirl** [opengisch/QField#4296] [Q&A]
I have newly mobile-device collected field data that needs to be pushed to my machine from QField. I pushed as normal without incident but when I try to synchronize this from the cloud on to my machine, nothing happens. Oddly, i get the photos taken in the field, but the main geopackage layer I need updated with field data just remains the same. 

When I go to my QField account, it says the most recent versions of data are from days ago, not the stuff I collected today. Is this maybe because I am a free tier member and it exceeds the hosting limits? I'm trying to delete all the imported photos so I can just keep the geopackage data, but it seems I cannot delete my photo data from my QField account.

I really need this data on my computer. Any help appreciated.

**Comment by Bvpraagh** [comment]
Sorry not answer but I am  having the same issue. All photos pushed and synchronized but not Geopackage layers.  I also thought it may be due to data amount but still room. Also tried deleting photos to make room but can't delete them from Cloud. Wondered if it is a bug of some sort.

**Reply by jsmith2-wirl** [reply]
@Bvpraagh Thanks for your comment. I noticed if I log into my cloud account and click a tab called "Changes", there's a list of "pending" changes. When I click the job ID the log of changes seems to correspond with what I was attempting to push. Sounds like this is held in queue until you pay for more storage or the opportunity to have more version controls. Have you checked that?

Would be nice for a developer to confirm if my interpretation of this is correct.

**Comment by Bvpraagh** [comment]
Yes same here. All  pending. However the total of my project files only adds up to 700 mb yet it says I have used over 900 mb. theoretically I should have plenty of room left for the small geopackage files to go through.  I think your interpretation is most likely correct even though I haven’t actually used the 1 gb storage.
